scrapy_crawlera/middleware.py

```diff
@@ -283,7 +283,7 @@ def _get_url_domain(self, url):
         return parsed.netloc
 
     def _is_crawlera_response(self, response):
-        return bool(response.headers.get("X-Crawlera-Version"))
+        return self._has_crawlera_headers(response)
```

Comment:
`_has_crawlera_headers` is O(n) while current is O(1). I think we should stick to:


```suggestion
        return bool("X-Crawlera-Version" in response.headers)
```

Reply:
Done